releases/release_notes_09_2022.md

```diff
@@ -0,0 +1,282 @@
+```
+kubectl get runtimeclass
+```
+Output: 
+```
+NAME            HANDLER         AGE
+kata            kata            9m55s
+kata-clh        kata-clh        9m55s
+kata-clh-tdx    kata-clh-tdx    9m55s
+kata-qemu       kata-qemu       9m55s
+kata-qemu-tdx   kata-qemu-tdx   9m55s
+```
+
+Details on each of the runtime classes:
+
+-- kata - standard kata runtime using the QEMU hypervisor including all COCO building blocks for a non CC HW
+-- kata-clh - standard kata runtime using the cloud hypervisor including all COCO building blocks for a non CC HW
+-- kata-clh-tdx - using the Cloud Hypervisor, with TD-Shim, and support for Intel TDX CC HW
+-- kata-qemu - same as kata
+-- kata-qemu-tdx - using QEMU, with TDVF, and support for Intel TDX CC HW
+-- * *TBD: we need to add the SEV runtimes as well* *
+
+
+# Post installation configuration 
+* *TBD:...* *
+
+# Creating a workload 
+## Creating a sample COCO workload 
+
+The first workload we create will show how the COCO building blocks work together without encryption or CC HW support (which will be demonstrated in later workloads). 
+
+A key point when working on COCO is to ensure that the container images get downloaded inside the VM and not on the host.
+
+In our example we will be using the bitnami/nginx image as described in the following yaml:
+```
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    run: nginx
+  name: nginx
+spec:
+  containers:
+  - image: bitnami/nginx:1.22.0
+    name: nginx
+  dnsPolicy: ClusterFirst
+  runtimeClassName: kata
+```
+
+For verifying that the container image doesn’t exist on the host you should log into the k8s node and ensure the following command returns an empty result:
+```
+root@cluster01-master-0:/home/ubuntu# crictl  -r  unix:///run/containerd/containerd.sock image ls | grep bitnami/nginx
+```
+
+Create a pod yaml file as previously described (we named it nginx.yaml) 
+
+Create the workload: 
+```
+kubectl apply -f nginx.yaml
+```
+Output: 
+```
+pod/nginx created
+```
+
+Ensure the pod was created successfully (in running state):
+```
+kubectl get pods        
+```
+Output: 
+```
+NAME    READY   STATUS    RESTARTS   AGE
+nginx   1/1     Running   0          3m50s
+```
+
+Now go back to the k8s node and ensure that you still don’t have any bitnami/nginx images on it:
+```
+root@cluster01-master-0:/home/ubuntu# crictl  -r  unix:///run/containerd/containerd.sock image ls | grep bitnami/nginx
+```
+
+## Creating a COCO workload using a pre-existing encrypted image
+
+We will now proceed to download and run an encrypted container image using the COCO building blocks. 
+
+* *TBD: based on https://github.com/confidential-containers/operator/issues/77* *
+
+
+## Creating a COCO workload using a pre-existing encrypted image on CC HW
+
+For running one of the sample workloads provided in the previous step, but now taking advantage of a specific TEE vendor, the user will have to set the runtime class of the workload accordingly in the workload yaml file.
```

Comment:
Nit: Could you fold these long lines to make diffs easier?

Reply:
Ack.
This section will be rewritten later on to include a lot more information on how to configure TDX and SEV-ES so we can change it then (this part was added by @fidencio )